src/GitTfs/Commands/RemoteOptions.cs

```diff
@@ -16,6 +16,10 @@ public OptionSet OptionSet
                         v => IgnoreRegex = v },
                     { "except-regex=", "A regex of exceptions to '--ignore-regex'",
                         v => ExceptRegex = v},
+                    { "gitignore:", "Use .gitignore to ignore files on download from TFS. Alternatively, provide path toward the .gitignore file which will be used to ignore files",
```

Comment:
It should be "gitignore="

Reply:
No, it should be as it is, making --gitignore path value optional, as https://github.com/git-tfs/git-tfs/pull/1214/commits/714ad615b2446da789b510916d4abb8f40f7334f describes in details ;)

Reply:
You're right. I forgot the NDesk syntax.